docs/design/hbar-limiter.md

```diff
@@ -271,6 +275,83 @@ c. Current day's usage (increase limits if overall usage is low)
    - Keep a small portion of the daily budget (e.g., 10%) as a reserve
    - Use this to accommodate unexpected spikes or high-priority users
 
+## Configurations
+
+### Pre-populating the Redis Cache with Spending Plans for Supported Projects and Partner Projects
+
+The following configurations will be used to automatically populate the Redis cache with `HbarSpendingPlan`, `EthAddressHbarSpendingPlan`, and `IPAddressHbarSpendingPlan` entries for the outlined supported projects and partner projects on every start-up of the relay.
+
+All other users (ETH and IP addresses which are not specified in the configuration file) will be treated as "general users" and will be assigned a basic `HbarSpendingPlan` on their first request and their ETH address and IP address will be linked to that plan for all subsequent requests.
+
+```json
+[
+  {
+    "name": "partner name",
+    "ethAddresses": ["0x123", "0x124"],
+    "ipAddresses": ["127.0.0.1", "128.0.0.1"],
+    "subscriptionTier": "ADVANCED"
+  },
+  {
+    "name": "some other partner that has given us only eth addresses",
+    "ethAddresses": ["0x125", "0x126"],
+    "subscriptionTier": "ADVANCED"
+  },
+  {
+    "name": "supported project name",
+    "ethAddresses": ["0x127", "0x128"],
+    "ipAddresses": ["129.0.0.1", "130.0.0.1"],
+    "subscriptionTier": "EXTENDED"
+  },
+  {
+    "name": "some other supported project that has given us only ip addresses",
+    "ipAddresses": ["131.0.0.1", "132.0.0.1"],
+    "subscriptionTier": "EXTENDED"
+  }
+]
+```
+
+On every start-up, the relay will check if these entries are already populated in the Redis cache. If not, it will populate them accordingly. 
+
+The JSON file can also be updated over time to add new supported projects or partner projects and it will populated only the new entries on the next start-up.
+
+```json
+[
+  ...,
+  {
+    "name": "new partner name",
+    "ethAddresses": ["0x129", "0x130"],
+    "ipAddresses": ["133.0.0.1"],
+    "subscriptionTier": "ADVANCED"
+  }
+]
+```
+
+### Spending Limits of Different Tiers
+
+The spending limits for different tiers are defined as environment variables:
+- `HBAR_RATE_LIMIT_BASIC`: The spending limit (in hbars) for general users (tier 3)
+- `HBAR_RATE_LIMIT_EXTENDED`: The spending limit (in hbars) for supported projects (tier 2)
+- `HBAR_RATE_LIMIT_PRIVILEGED`: The spending limit (in hbars) for trusted partners (tier 1)
+
+Example configuration for tiered spending limits:
+```dotenv
+HBAR_RATE_LIMIT_BASIC=1000
+HBAR_RATE_LIMIT_EXTENDED=10000
+HBAR_RATE_LIMIT_PRIVILEGED=20000
+```
+
+### Total Budget and Limit Duration
+
+The total budget and the limit duration are defined as environment variables:
+- `HBAR_RATE_LIMIT_DURATION`: The time window (in milliseconds) for which both the total bud
+- `HBAR_RATE_LIMIT_TINYBAR`: The ceiling on the total amount of HBars that can be spent in the limit duration.
```

Comment:
Given the context above we should be explicit and note this is the largest bucket from which others pull from.